### PR TITLE
Upstream  0.13.8 + rest of R19

### PR DESCRIPTION
**Pull request by motobob**
upstream changes + All R19 compatibility spec fixes

### CODE REVIEW

**Comment by motobob** [comment]
this one is to substitute #115 too

File: src/erlcloud.app.src [hunk 1]
```diff
@@ -12,6 +12,7 @@
                   ssl,
                   xmerl,
                   inets,
+                  eini,
```
Comment (motobob) [inline src/erlcloud.app.src]: manual merge error. to be removed

**Comment by motobob** [comment]
test it

**Comment by motobob** [comment]
@nalundgaard @noproc please take a look and merge

**Comment by nalundgaard** [comment]
Holy big change, batman!

Looks like this is mostly the spec fixes for 19 and some very minor other new additions. I tested this with our internal functional testing for SES and DDB, and it looks good, so I am +1 there. I don't deal with any SW that touches the assumeRole stuff, but it looks like one area where there are some changes. Have you guys verified compatibility there?

Other than that, I am 👍 on this.

**Comment by motobob** [comment]
tested in real stack - ok.
tested the profile loading too - it works too. 

shipping it

**Pull request merged**